documentation/modules/proc-purging-queues.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// assembly-.adoc
+
+[id='proc-purging-queues-{context}']
+= Purging queues and subscriptions
+
+You can purge a `queue` or `subscription` address type that contains stored messages using the {ConsoleName}.
```

Comment:
```suggestion
You can purge a `queue` or `subscription` address type that contains stored messages by using the {ConsoleName}.
```

Reply:
It reads awkwardly to me.   How about "You can purge a `queue` or `subscription` address type of its stored messages by using the {ConsoleName}"